libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/NativeLoginManager.kt

```diff
@@ -673,6 +770,78 @@ internal class NativeLoginManager(
         val result: NativeLoginResult? = null
     )
 
+    /**
+     * Submits an authorization request to the Salesforce Identity API and, on
+     * success, submits the access token request.
+     * @param authRequestType The Salesforce Identity API authorization request
+     * type header value
+     * @param otp A one-time-password (OTP) previously issued by the Salesforce
+     * Identity API
+     * @param identifier A OTP or request identifier previously issued by the
+     * Salesforce Identity API to match the provided OTP
+     * @param otpVerificationMethod The OTP verification method used to obtain
+     * the identifier from the Salesforce Identity API
+     * @return A native login result indicating success or one of several
+     * possible failures, including both in-app and Salesforce Identity API
+     * results
+     */
+    private suspend fun submitAuthorizationRequest(
```

Comment:
This code should be 99.9% identical to what's on `dev` with the exception of the `authRequestType` parameter.